### DISCUSSION TITLE
find a bug

### DISCUSSION THREAD

**Discussion by jxlongyuan** [community/community#122873] [Programming Help]
###  InfluxDB QueryApiSync BUG

Bug

### Body

today , i was used   [https://github.com/influxdata/influxdb-client-csharp](https://github.com/influxdata/influxdb-client-csharp/)   queryApiSync -> func  QuerySync    to action  query InfluxDb ，un used the org  parameter  was faild , used the org parameter   result success ,document say this org parameter can by empty ,but org parameter  must not empty, test result,the org parameter  enter anything values all success , please test this func .

**Comment by 22coffeeboi** [comment]
It sounds like you've encountered a bug with the QueryApiSync function in the InfluxDB client for C#. From your description, it seems that the org parameter, which according to the documentation can be empty, is actually required for the query to succeed.

Here are a few steps you can take to address this issue:

Review Documentation: Double-check the documentation for the QueryApiSync function to ensure that you're using it correctly. If the documentation states that the org parameter can be empty, but your testing indicates otherwise, there may be an inconsistency that needs to be addressed.
Verify Function Implementation: Review the source code of the QueryApiSync function in the InfluxDB client for C# to understand how it handles the org parameter. There might be a bug or oversight in the implementation that causes it to fail when the org parameter is empty.
Test Different Input Values: As you mentioned, try different values for the org parameter to see if there's a specific value format or requirement that needs to be met for the query to succeed. This could help you understand the behavior of the function and identify any patterns or inconsistencies.
Report the Issue: If you're confident that you've identified a bug in the QueryApiSync function, consider reporting it to the maintainers of the InfluxDB client for C#. You can open an issue on the GitHub repository or reach out through other channels to report the bug and provide details of your findings.
Workaround: If possible, try to find a workaround for the issue while waiting for a fix from the maintainers. This could involve providing a non-empty value for the org parameter or using a different approach to querying InfluxDB that doesn't rely on the QueryApiSync function.
By following these steps, you should be able to investigate and address the issue with the QueryApiSync function in the InfluxDB client for C#
